### PR TITLE
Output Added

### PR DESCRIPTION
**Pull request by Dustin1230**
Added a commented section for the hook files that shows the output of the PseudoCode changes.

### CODE REVIEW

File: Puesdo Hooks/XComGame/XGAbility/ApplyCost.txt
Comment (AzXeus) [inline Puesdo Hooks/XComGame/XGAbility/ApplyCost.txt:1]: output comment doesn't match psuedocode

File: Puesdo Hooks/XComGame/XGAbility/ShowAbility.txt
Comment (AzXeus) [inline Puesdo Hooks/XComGame/XGAbility/ShowAbility.txt:1]: output comment doesn't match psuedocode

File: Puesdo Hooks/XComGame/XGUnit/AbsorbDamage.txt
Comment (AzXeus) [inline Puesdo Hooks/XComGame/XGUnit/AbsorbDamage.txt:1]: output comment doesn't match psuedocode

File: Puesdo Hooks/XComGame/XGUnit/ApplyInventoryStatModifiers.txt
Comment (AzXeus) [inline Puesdo Hooks/XComGame/XGUnit/ApplyInventoryStatModifiers.txt:1]: output comment doesn't match psuedocode

File: Puesdo Hooks/XComGame/XGUnit/OnTurnBeginHook.txt
Comment (AzXeus) [inline Puesdo Hooks/XComGame/XGUnit/OnTurnBeginHook.txt:1]: output comment doesn't match psuedocode

**Review by AzXeus — CHANGES_REQUESTED** [review]

**Comment by Dustin1230** [comment]
Made the changes for ya

**Review by AzXeus — APPROVED** [review]
all looks good

**Pull request merged**